src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -90,3 +82,13 @@ set_target_properties(System.Globalization.Native-Static PROPERTIES PREFIX "")
 set_target_properties(System.Globalization.Native-Static PROPERTIES OUTPUT_NAME System.Globalization.Native  CLEAN_DIRECT_OUTPUT 1)
 
 install (TARGETS System.Globalization.Native-Static DESTINATION .)
+
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
```

Comment:
It will still run on non-OSX (the condition is `if not Darwin`). The reason why i moved this block down is, when I updated the minimum version of cmake in top `CMakeLists.txt` and started to use `configuretools.cmake`, it started to give me fatal error:

```
  CMake Error at System.Globalization.Native/CMakeLists.txt:44 (add_custom_command):
    No TARGET 'System.Globalization.Native' has been created in this directory.
```

the target gets defined on line 62 (in this revesion).